src/modules/75-cf/strings/strings.en.yaml

```
@@ -519,6 +519,11 @@ gitSync:
   goToGit: Go to Git to fix issue
   invalidYaml: 'We are unable to render your {{{ flagsYamlFilename }}} file because it is invalid. Fix the following issues on Git and try again.'
   setUpGitConnection: Set up a Git connection
+  setUpGitSuccess: 'Successfully connected to repository {{ repoName }}'
+  resetGitSettings: Reset Git Settings
+  resetGitWarning: This will delete all git configurations and you will have to set it up from the beginning
+  resetGitSuccess: Git Sync reset successfully
+  resetGitError: Error resetting Git Sync
```

Do we want to still say `Git Sync`? Maybe `Git settings` to keep in line with the action?
```suggestion
  resetGitSuccess: Git settings reset successfully
  resetGitError: Error resetting git settings
```

---

Done